cli/tests/integrations/helpers/common.py

```diff
@@ -204,9 +204,9 @@ def update_config(name, value, env=None):
     # we need to also restore the token if this occurs
     token = None
     if name == "core.dcos_url":
-        returncode, token_val, _ = exec_command(
+        returncode_, token_val, _ = exec_command(
             ['dcos', 'config', 'show', "core.dcos_acs_token"], env)
-        if returncode == 0:
+        if returncode_ == 0:
```

Comment:
Oh I see. It was clobbering the previous value of `returncode` so if you don't have a token, it thinks you have no config and doesn't try to update it.
👍